source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
+   For some categories, it can be possible to select more than one value. In this case, after clicking on a value, the others do not disappear. Click on any other value to filter even more.
 
-Filters
-~~~~~~~
+   For other categories, such as "Modification", the values are sorted in a hierarchical way. It means that the year must be selected first, then the months are displayed. Once the month is selected, the days are displayed in order to be selected as well.
 
-Under the geographic search map, there is a list of selectable filters sorted by category. They are constructed from the datasets metadata and enable the user to filter all the datasets that have the same attributes like language, theme, keywords, publisher, date, among others.
+Downloading the catalog
+-----------------------
 
-.. image:: images/filters_facets.png
-   :alt: Selectable filters
+Under all the filters are different possibilities to download the catalog. This does not allow to download all of the datasets from the catalog (or the filtered ones), but it allows to download an index of the datasets from the catalog along with their main metadata.
 
-Click on the values you want to use to filter the dataset list. For some categories
-like keywords and themes, it is possible to select multiple values, and for some
-others like modification date, the selection is hierarchical. For example,
-selecting a year will let you to choose the month.
+.. image:: images/download_catalog.png
+   :alt: Possible formats to download the catalog
 
-In the case of keywords, you can also click on them directly on a dataset card to
-filter the catalog on that keyword.
+It is possible to download the catalog in the following formats:
+
+- XLS
+- CSV
+- RDF (Turtle)
+- RDF (XML)
+- RSS feed
+
+Exploring a dataset from the catalog
+------------------------------------
+
+Once the right dataset of the catalog has been found, through the searching and filtering options or by scrolling down, it should be time to explore this dataset to see the data and their visualizations.
+
+.. image:: images/catalog_card.png
+   :alt: Catalog card of a dataset from a catalog
+
+From the catalog interface, only the catalog card of the dataset is displayed. The catalog card should show the following information:
+
+- a pictogram, to represent the theme of the dataset
+- the title of the dataset
+- the description of the dataset
+- the name of the publisher
+- the license of the dataset
+- the keywords of the dataset
+- links to the available visualizations of the dataset (e.g. Table, Map, etc.)
+- links to tabs to share or reuse the dataset (e.g. Export, API)
 
 .. admonition:: Note
    :class: note
 
-   The catalog filters of a portal can be customized, including which filters are proposed, the order of the filters, and how to order the suggested values. Please contact us for more information.
+   Less information can be displayed: it depends on which metadata have been set before the dataset was published.
```

Comment:
Also, you should state the the catalog card can be fully customized by the portal editor.